java/src/jmri/jmrix/cmri/serial/nodeconfigmanager/NodeConfigManagerFrame.java

```diff
@@ -821,7 +821,7 @@ public void initNodeConfigWindow() {
         nodeTypeBox.addItem("SMINI");
         nodeTypeBox.addItem("USIC_SUSIC");
         nodeTypeBox.addItem("CPNODE");
-
+        nodeTypeBox.addItem("CPMEGA");
         //  Hide the menu item until MRCS are ready to release the cpMega
```

Comment:
Please remove these comments indicating the CPMEGA should be hidden.

Reply:
removed hide statment

Reply:
GitHub is showing it was not removed (the way you attempted to remove it created a PR that conflicts with this PR). Please remove it by editing the file at this link: https://github.com/adeak1rr/JMRI/blob/patch-1/java/src/jmri/jmrix/cmri/serial/nodeconfigmanager/NodeConfigManagerFrame.java